include/discord-internal.h

```diff
@@ -28,6 +28,7 @@ extern "C" {
 #include "io_poller.h"
 #include "queue.h"
 #include "priority_queue.h"
+#include "queriec.h"
```

Comment:
I'd say its better to move this include to the actual `.c` files that use it: `audit_log.c, channel.c, guild.c, guild_scheduled_event.c, interaction.c, webhook.c`